### DISCUSSION TITLE
Two Different projects show two different times

### DISCUSSION THREAD

**Discussion by TajShireen** [moment/luxon#1555] [Q&A] [answered]
Hi, I have a peculiar problem. I am using Luxon with Sveltekit and with a Svelte-only project. 

When I run time `2025-09-30T09:30:00.000Z` through the function 
```
		DateTime.fromISO(date).toFormat("hh:mm");
```
and set my browser timeZone to Alaska, in the SvelteKit project I get the time as "1:30" AM and in the Svelte project I get the time "12:30" AM. I am very confused as to why this is happening. Google says 12:30 is the accurate one.

**Comment by diesieben07** [comment, marked as answer]
Can you share the output of `DateTime.fromISO(date).resolvedLocaleOptions()` in both projects? Or, ideally, a way to reproduce this?

**Reply by TajShireen** [reply]
Hey, I was able to solve it by upgrading node for both projects. Thank you for your comment.
